Dockerfile

```diff
@@ -60,28 +60,9 @@ RUN \
     echo "===> install Java"  && \
     echo debconf shared/accepted-oracle-license-v1-1 select true | debconf-set-selections  && \
     echo debconf shared/accepted-oracle-license-v1-1 seen true | debconf-set-selections  && \
-    DEBIAN_FRONTEND=noninteractive  apt-get install -y --force-yes libnss3 libgconf-2-4 libxss1 libxtst6 libgtk2.0-0 libasound2 xvfb netcat oracle-java8-installer oracle-java8-set-default
-
-RUN mkdir -p /application
-
-WORKDIR /application
-
-# Create empty directory for selenium logs
-
-RUN mkdir -p logs/selenium
-
-# Install required npm dependencies
-
-COPY package.json .
-COPY yarn.lock .
-COPY .yarnrc .
-# skips all dev dependencies if NODE_ENV=production.. so..
-RUN yarn install --production=false
-
-# Copy application source to image
+    DEBIAN_FRONTEND=noninteractive  apt-get install -y --force-yes libnss3 libgconf-2-4 libxss1 libxtst6 libgtk2.0-0 libasound2 xvfb netcat oracle-java8-installer oracle-java8-set-default && \
```

Comment:
Setting this env var is actively discouraged by docker: https://docs.docker.com/engine/faq/ unless it's causing an error with the build I think we should drop it.

Reply:
Since we're not exporting this as an image for any downstream use the concern in that FAQ does not really apply. I'm not opposed to removing it, or using a node docker image, but given how many vets-website builds happen I'm in favor of incremental improvements and maybe a separate ticket (if there's not one already) for overall build optimization.